### PR TITLE
Added link further explaining ERB in the 'Getting Started' documentation [ci skip]

### PR DESCRIPTION
**Pull request by mguan2020**
### Motivation / Background

This Pull Request has been created because it would be very helpful to provide a link that explains ERB in more depth for people learning the Rails framework. ERB is a very essential part of a Rails application.

### Detail

This Pull Request adds the important fact that ERB stands for "Embedded Ruby." This helps readers better understand the meaning of ERB. This Pull Request also provides a link to a very useful Wikipedia page explaining ERB in more depth for readers who want to explore the topic in more depth.

### Checklist

Before submitting the PR make sure the following are checked:

* [X] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [X] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [X] Tests are added or updated if you fix a bug or add a feature.
* [X] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -558,7 +558,7 @@ file, and replace its contents with:
 </ul>
 ```
 
-The above code is a mixture of HTML and *ERB*. ERB is a templating system that
+The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby#:~:text=Embedded%20Ruby%20), is a templating system that
```
Comment (akhilgkrishnan) [inline guides/source/getting_started.md:561]:
```suggestion
The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby), is a templating system that
```

Comment (mguan2020) [reply guides/source/getting_started.md:561]: Great suggestion! I'll go ahead and make the change.

**Review by akhilgkrishnan — COMMENTED** [review]
Could you please do a [squash commit](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits) and amend the commit message with `[skip ci]`

**Comment by mguan2020** [comment]
Hi @akhilgkrishnan,

I am having some issues with squashing my commits. I am getting the error "error: failed to push some refs to 'https://github.com/mguan2020/rails.git" after I run the command 'git push fork branch_new --force-with-lease'.
I have carefully read the guide on Contributing to Ruby on Rails.

Would it be possible for you to help? Thank you very much for your time.

**Comment by akhilgkrishnan** [comment]
> Hi @akhilgkrishnan,
> 
> 
> 
> I am having some issues with squashing my commits. I am getting the error "error: failed to push some refs to 'https://github.com/mguan2020/rails.git" after I run the command 'git push fork branch_new --force-with-lease'.
> 
> I have carefully read the guide on Contributing to Ruby on Rails.
> 
> 
> 
> Would it be possible for you to help? Thank you very much for your time.

Seems like you haven't taken the latest pull. Take the pull first and then do the squashing.

**Comment by mguan2020** [comment]
Hi @akhilgkrishnan,

Thank you for your advice. I have squashed the commits into one single commit.

Please let me know if there is anything else I need to do.

**Comment by akhilgkrishnan** [comment]
> Hi @akhilgkrishnan,
> 
> 
> 
> Thank you for your advice. I have squashed the commits into one single commit.
> 
> 
> 
> Please let me know if there is anything else I need to do.


I think the [skip ci] still missing on your commit message. So could you please amend your commit.

**Comment by mguan2020** [comment]
> > Hi @akhilgkrishnan,
> > Thank you for your advice. I have squashed the commits into one single commit.
> > Please let me know if there is anything else I need to do.
> 
> I think the [skip ci] still missing on your commit message. So could you please amend your commit.

Hi @akhilgkrishnan,

Thank you for the reminder. I have appended [skip ci] to my commit message.

**Comment by akhilgkrishnan** [comment]
> > > Hi @akhilgkrishnan,
> > > Thank you for your advice. I have squashed the commits into one single commit.
> > > Please let me know if there is anything else I need to do.
> > 
> > 
> > I think the [skip ci] still missing on your commit message. So could you please amend your commit.
> 
> Hi @akhilgkrishnan,
> 
> Thank you for the reminder. I have appended [skip ci] to my commit message.

I don't see it 🤔

**Comment by mguan2020** [comment]
> > Hi @akhilgkrishnan,
> > Thank you for the reminder. I have appended [skip ci] to my commit message.
> 
> I don't see it 🤔

Apologies for the confusion. It should be there now.

**Comment by akhilgkrishnan** [comment]
> > > Hi @akhilgkrishnan,
> > > Thank you for the reminder. I have appended [skip ci] to my commit message.
> > 
> > 
> > I don't see it 🤔
> 
> My apologies for the confusion. It should be there now.

No apologies please.

**Review by akhilgkrishnan — APPROVED** [review]
Thanks @mguan2020 👍

**Comment by zzak** [comment]
@akhilgkrishnan We are not particularly strict about adding `ci skip` to doc commits, fyi ❤️

**Review by zzak — APPROVED** [review]
The line wrapping is throwing me off but I guess it's fine :joy:

File: guides/source/getting_started.md [hunk 2]
```diff
@@ -558,7 +558,7 @@ file, and replace its contents with:
 </ul>
 ```
 
-The above code is a mixture of HTML and *ERB*. ERB is a templating system that
+The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby), is a templating system that
```
Comment (rafaelfranca) [inline guides/source/getting_started.md:561]:
```suggestion
The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://docs.ruby-lang.org/en/3.2/ERB.html), is a templating system that
```

**Comment by akhilgkrishnan** [comment]
@mguan2020 Could you please update the suggested changes?

**Comment by mguan2020** [comment]
> @mguan2020 Could you please update the suggested changes?

Thank you very much for the reminder. I have incorporated @rafaelfranca suggested changes. All of the commits are now squashed into one.

Could you please review the PR again? Thank you for your time.

**Comment by akhilgkrishnan** [comment]
> > @mguan2020 Could you please update the suggested changes?
> 
> Thank you very much for the reminder. I have incorporated @rafaelfranca suggested changes. All of the commits are now squashed into one.
> 
> Could you please review the PR again? Thank you for your time.

Looks good to me 👍

**Pull request merged**